decidim-forms/app/packs/src/decidim/forms/admin/forms.js

```diff
@@ -11,6 +11,7 @@ import createSortList from "src/decidim/admin/sort_list.component"
 import createDynamicFields from "src/decidim/admin/dynamic_fields.component"
 import createFieldDependentInputs from "src/decidim/admin/field_dependent_inputs.component"
 import createQuillEditor from "src/decidim/editor"
+import initLanguageChangeSelect from "src/decidim/admin/choose_language"
```

Comment:
I agree overall and understood the point.
But this particular js `src/decidim/forms/admin/forms.js` is sitting under the `/admin/` path in `deciidm-forms`, and is loaded for admin panel only, right? Which doesn't make sense without the `decidim-admin` module included.
This function is actually a part of `decidim-admin` as it applies to specific select elements displayed in admin panel and also rely on specific elements structure there.
Moreover, there is a bunch of other functions imported from admin scripts here.

I'm not willing to avoid that refactoring, maybe please double check if it's really necessary.

Reply:
Sure, I see your point. This particular language selector is tied to the admin module interface, so it is strongly related to the admin module.

So I agree after your good reasoning. We should keep it in the admin module and refer to it from there in the forms admin JS.

Thanks very much putting thought into this and correcting my review mistake! My initial thinking was just that general code should live in `decidim-core` but I didn't realize this is actually pretty strongly related to the admin panel and has no use outside of it.